pyo3-derive-backend/src/pyclass.rs

```diff
@@ -92,20 +92,25 @@ impl PyClassArgs {
                 self.freelist = Some(syn::Expr::clone(right));
             }
             "name" => match &**right {
-                syn::Expr::Lit(
-                    lit
-                    @
-                    syn::ExprLit {
-                        lit: syn::Lit::Str(..),
-                        ..
-                    },
-                ) => {
-                    self.name = Some(lit.clone().into());
+                syn::Expr::Lit(syn::ExprLit {
+                    lit: syn::Lit::Str(lit),
+                    ..
+                }) => {
+                    self.name = Some(lit.parse()?);
```

Comment:
Maybe we should catch the error from `parse` here, in order to display a friendlier error.

Reply:
Agreed, pushed a fix.